stdlib/public/runtime/Casting.cpp

```diff
@@ -44,6 +44,11 @@
 #include <cstring>
 #include <type_traits>
 
+#if __GLIBCXX__-0 < 20160726
```

Comment:
Won't this be true when not using glibc?

Reply:
Ah, crap.  I guess I can't do the clever trick.  I'll change this to

    #if defined(__GLIBCXX__) && __GLIBCXX__ < 20160726

Good catch!